.dockerignore

```diff
@@ -45,6 +45,7 @@
 # Add tests and kubernetes_tests to context.
 !tests
 !kubernetes_tests
+!chart_tests
```

Comment:
Anything stopping us using chart/tests like we had before, just with python files?

Reply:
^^ @dimberman

Reply:
moved to chart/tests

Reply:
@ashb @kaxil moving the files to `/chart` caused mypy to fail

```
Run mypy..........................................................................................................................Failed
- hook id: mypy
- exit code: 2

tests/__init__.py: error: Duplicate module named 'tests' (also at
'chart/tests/__init__.py')
tests/__init__.py: error: Are you missing an __init__.py?
Found 2 errors in 1 file (checked 2459 source files)
```

So I guess our options are either create a `chart_tests` directory, change the name of the testing directory within the chart, or add an __init__.py  to the chart directory (unless y'all have another route we can take). Between those three choices I'd rather the `chart_tests` because I don't like the idea of putting an __init__.py into a helm chart for testing purposes. WDYT?

Reply:
We can also run mypy two times - one for core, one for Helm Chart.